book/07-git-tools/sections/bundling.asc

```diff
@@ -1,20 +1,20 @@
 [[_bundling]]
-=== Bundling
+=== Demetleme (Bundling)
```

Comment:
Bundle kelimesine o kadar alismisim ki hic Turkce anlami ne olur diye dusunmemisim :D Demetleme guzel fikirmis.

Reply:
Teşekkürler hocam :)
Tüm bölümlerin özet ve giriş kısımlarını çevirdim ama bir sebepten dolayı PR kabul edilemiyor. Sebebi hakkında fikriniz var mı?

Reply:
icerikle ilgili sorun -cogu zaman- yok, fakat bazen paralel branch'lerde duplicate commit'ler arasinda kayboldugum oluyor. 

soyle yapalim, bunu tamamlayip merge edelim. sonra master'dan yeni bir branch alip ordan devam edelim. 239 commit okumak ciddi vakit aliyor yoksa. o sekilde 15-20'lere indiririz diye dusunuyorum.

Reply:
Aslında niyetim her sayfayı tek commit ile tamamlamak ve her bölüm sonunda PR çekmek. Ama çeviriyi genellikle işe gidip gelirken trende yaptığım için, aktarma vs yaparken bilgisayarı kapatmadan önce yaptıklarımı kaydetmek için commitliyorum. Tabi okuyan kişi gözünden görmediğim için bunun sizin işinizi zorlaştırıcı bir etken olduğunu farketmemiştim. Bundan sonra daha az commit ile işi bitirmeye çalışırım.

Reply:
Hocam benim bu çeviri işine girmem Git'i detaylı öğrenme gerekliliğimle başladı. Klavuzun Slovence çevirisi bile varken Türkçe çevirisi olmaması zoruma gitti. Biraz da başkasına anlatamadığın bilgiye hakim değilsindir mottosuyla kendimi gaza getirip başladım. 

Hem kılavuzu çevirirken, hem de GitHub üzerinden süreci yürütürken çok şey ögrendim. Halen de öğrenmeye devam ediyorum. Bunun dışında, Git'in kullanmadığım ve bu yüzden detaylı bilmediğim pek çok özelliği de var. Hatta bu nedenle bazı yerlerde bazı kavramları düzgün ifade edememiş olabilirim. 

Onun dışında meseleyi sizlerin gözünden de görmediğim için gözden kaçırabileceğim iyi uygulama örnekleri olabilir. Sizden ricam hem çeviri (Türkçeleştirme vs) hem de daha ideal bir Git kullanıcı olma yolunda bana açık yüreklilikle tavsiyelerde bulunmanız. 40 Yaşında sektöre girmiş biri olarak, tecrübeli gençlerden öğrenecek çok şeyim olduğunu düşünüyorum.

Bu arada yolunuz Düsseldorf tarafına düşerse size kahve ısmarlamak isterim.